tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -142,15 +142,22 @@ USE_PODMAN          If set and USE_DOCKER not set, then build the rootfs inside
                     a podman container (requires podman).
                     Default value: <not set>
 
-SKOPEO_UMOCI        If set to "yes", build Skopeo and umoci for confidential
+SKOPEO              If set to "yes", build Skopeo for confidential
                     containers guest image pull. Currently, this is only
                     supported for Ubuntu guests; see
                     https://github.com/kata-containers/kata-containers/pull/2908
                     for discussion.
                     Default value: <not set>
+				
```

Comment:
sorry, had wanted to comment before update: can you remove these tabs?

Reply:
Done 👍